contracts/ETO/ETOCommitment.sol

```diff
@@ -554,7 +562,8 @@ contract ETOCommitment is
     {
         // add 1 to MIN_TICKET_TOKEN because it was produced by floor and check only MAX CAP
         // WHITELIST CAP will not induce state transition as fixed slots should be able to invest till the end of Whitelist
-        bool capExceeded = isCapExceeded(false, MIN_TICKET_TOKENS + 1, 0);
+        // also put the minimum ticket size plus one cent as eur equivalent to see wether we would cross the threshold
+        bool capExceeded = isCapExceeded(false, MIN_TICKET_TOKENS + 1, 0, ETO_TERMS.MIN_TICKET_EUR_ULPS() + 1**16);
```

Comment:
please cache `MIN_TICKET_EUR_ULPS` in local storage. it's additional 900 units of gas. 
also 1**16 = 1 ;> (that's why we test event the smallest thing)

also you do not need to add 10**16. if `MIN_TICKET_EUR_ULPS` can still be invested then it's OK, adding any value would prohibit that